wallet/db.c

```diff
@@ -270,6 +270,17 @@ char *dbmigrations[] = {
     /* Record the msatoshi actually sent in a payment. */
     "ALTER TABLE payments ADD msatoshi_sent INTEGER;",
     "UPDATE payments SET msatoshi_sent = msatoshi;",
+    /* -- Node statistics -- */
+    "CREATE TABLE node_statistics"
+    "     ( id INTEGER PRIMARY KEY AUTOINCREMENT"
+    "     , nodeid BLOB UNIQUE"
```

Comment:
I've always used the commas as the last element on the line.

Reply:
I prefer the commas as the first element, since the usual modification is to add new items at the end of sequence.

Consider a sequence like the below:

    FOO A,
        B

When we add C, we also modify the previous line for B even if in principle it should not need modification:

    FOO A,
        B,
        C

But with parentheses at the start of lines:

    FOO A
      , B

Then we add a new C, the line for B is unmodified:

    FOO A
      , B
      , C